### PR TITLE
Fix SD-Card, change the menu structure better to look more nicer, root mode now keep the session and dont quit, adding translation page, update baout screen....

### PR DESCRIPTION
**Pull request by ruditimmermans**

### CODE REVIEW

File: qml/pages/About.qml [hunk 1]
```diff
@@ -30,6 +30,15 @@ Page {
 
         contentHeight: column.height + header.height + Theme.paddingLarge
 
+        PullDownMenu
```
Comment (Michal-Szczepaniak) [inline qml/pages/About.qml]: I would really appreciate keeping the consistency of 
```Type {```

File: qml/pages/Places.qml [hunk 2]
```diff
@@ -30,11 +30,20 @@ Page {
...
+            MenuItem {
+                text: qsTr("About Nanofiles")
+                onClicked: pageStack.push(Qt.resolvedUrl("About.qml"))
+            }
             MenuItem {
                 text: qsTr("Reset entries")
                 onClicked: settings.places = defaultPlaces
             }
-
+            MenuItem {
```
Comment (Michal-Szczepaniak) [inline qml/pages/Places.qml]: Why move it here? Makes more sense for it to be on main page no?

File: qml/pages/TranslationsPage.qml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        anchors.fill: parent
+        contentHeight: content.height
+
+        VerticalScrollDecorator {}
+
+        PullDownMenu
+        {
+            MenuItem
+            {
+                text: qsTr("POEditor translations")
```
Comment (Michal-Szczepaniak) [inline qml/pages/TranslationsPage.qml:17]: What is POEditor and why do we need it?

File: rpm/nanofiles.spec [hunk 4]
```diff
@@ -5,8 +5,8 @@ Name:       nanofiles
 %{!?qtc_make:%define qtc_make make}
 %{?qtc_builddir:%define _builddir %qtc_builddir}
 Summary:    nanofiles
-Version:    1.3.2
-Release:    1
+Version:    1.4
+Release:    0
```
Comment (Michal-Szczepaniak) [inline rpm/nanofiles.spec]: Keeping release 1 is standard i think?

File: src/fileengine.cpp [hunk 5]
```diff
@@ -195,12 +195,16 @@ bool FileEngine::changeFilePermission(QString fullPath, int permissionPos)
...
 {
-    if (QDir("/run/user/media/sdcard").exists())
-        return "/run/user/media/sdcard";
-    else if (QDir("/media/sdcard").exists())
-        return "/media/sdcard";
-    else
-        return "";
+    QString path = "";
+
+if (QDir("/run/media/defaultuser").exists())
```
Comment (Michal-Szczepaniak) [inline src/fileengine.cpp]: lacks indentation

**Review by Michal-Szczepaniak — CHANGES_REQUESTED** [review]

File: README.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
+<h1 align="center">
+  <br>
+  <img src="https://raw.githubusercontent.com/Michal-Szczepaniak/nanofiles/master/icons/172x172/nanofiles.png" width="256px" alt="Nanofiles">
+  <br>
+  <br>
+  Fileman
```
Comment (Michal-Szczepaniak) [inline README.md]: Fileman?

File: src/fileengine.cpp [hunk 7]
```diff
@@ -195,12 +195,16 @@ bool FileEngine::changeFilePermission(QString fullPath, int permissionPos)
...
+    path = "/run/media/defaultuser";
+else if (QDir("/run/media/nemo").exists())
+    path = "/run/media/nemo";
+else if (QDir("/media/sdcard").exists())
+    path = "/media/sdcard";
+
+return path;
```
Comment (Michal-Szczepaniak) [inline src/fileengine.cpp]:
```suggestion
    if (QDir("/run/media/defaultuser").exists())
        path = "/run/media/defaultuser";
    else if (QDir("/run/media/nemo").exists())
        path = "/run/media/nemo";
    else if (QDir("/media/sdcard").exists())
        path = "/media/sdcard";

    return path;
```

File: src/fileengine.cpp (hunk 5 above)
Comment (ruditimmermans) [reply src/fileengine.cpp]: When you remove the return ""; it would work anymore

File: rpm/nanofiles.spec (hunk 4 above)
Comment (ruditimmermans) [reply rpm/nanofiles.spec]: Ok i will change this

File: src/fileengine.cpp (hunk 5 above)
Comment (Michal-Szczepaniak) [reply src/fileengine.cpp]: i'm not talking about removing return.

File: src/fileengine.cpp (hunk 7 above)
Comment (ruditimmermans) [reply src/fileengine.cpp]: Yes i got it now i(m changing it...

File: qml/components/CollaboratorsLabel.qml [hunk 8]
```diff
@@ -0,0 +1,33 @@
+import QtQuick 2.5
+import Sailfish.Silica 1.0
+
+Column
+{
```
Comment (Michal-Szczepaniak) [inline qml/components/CollaboratorsLabel.qml]: These need to be changed 
```suggestion
Column {
```

File: qml/components/CollaboratorsLabel.qml [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+    property alias title: typelabel.text
+    property var labelData
+
+    id: collaboratorslabel
+    anchors { left: parent.left; right: parent.right }
+    visible: labelData.length > 0
+
+    Label
+    {
```
Comment (Michal-Szczepaniak) [inline qml/components/CollaboratorsLabel.qml]:
```suggestion
    Label {
```

File: qml/components/CollaboratorsLabel.qml [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+        id: typelabel
+        anchors { left: parent.left; right: parent.right }
+        horizontalAlignment: Text.AlignHCenter
+    }
+
+    Repeater
+    {
```
Comment (Michal-Szczepaniak) [inline qml/components/CollaboratorsLabel.qml]:
```suggestion
    Repeater {
```

File: qml/pages/About.qml [hunk 11]
```diff
@@ -30,6 +30,13 @@ Page {
 
         contentHeight: column.height + header.height + Theme.paddingLarge
 
+        PullDownMenu {
+        MenuItem {
+            text: qsTr("Nanofiles Translations")
+            onClicked: pageStack.push(Qt.resolvedUrl("TranslationsPage.qml"))
+        }
+     }
```
Comment (Michal-Szczepaniak) [inline qml/pages/About.qml]:
```suggestion
            MenuItem {
                text: qsTr("Nanofiles Translations")
                onClicked: pageStack.push(Qt.resolvedUrl("TranslationsPage.qml"))
            }
        }
```

File: qml/pages/TranslationsPage.qml [hunk 12]
```diff
@@ -0,0 +1,56 @@
+import QtQuick 2.5
+import Sailfish.Silica 1.0
+import "../components"
+
+Page
+{
```
Comment (Michal-Szczepaniak) [inline qml/pages/TranslationsPage.qml]:
```suggestion
Page {
```

File: qml/pages/TranslationsPage.qml [hunk 13]
```diff
@@ -0,0 +1,56 @@
...
+    id: developerspage
+        allowedOrientations: Orientation.All
+
+    SilicaFlickable
+    {
```
Comment (Michal-Szczepaniak) [inline qml/pages/TranslationsPage.qml]:
```suggestion
    SilicaFlickable {
```

File: qml/pages/TranslationsPage.qml [hunk 14]
```diff
@@ -0,0 +1,56 @@
...
+        anchors.fill: parent
+        contentHeight: content.height
+
+        VerticalScrollDecorator {}
+
+        PullDownMenu
+        {
```
Comment (Michal-Szczepaniak) [inline qml/pages/TranslationsPage.qml]:
```suggestion
        PullDownMenu {
```

File: qml/pages/TranslationsPage.qml [hunk 15]
```diff
@@ -0,0 +1,56 @@
...
+            MenuItem
+            {
```
Comment (Michal-Szczepaniak) [inline qml/pages/TranslationsPage.qml]:
```suggestion
            MenuItem {
```

File: qml/pages/TranslationsPage.qml [hunk 16]
```diff
@@ -0,0 +1,56 @@
...
+                text: qsTr("POEditor translations")
+                onClicked: Qt.openUrlExternally("https://poeditor.com/join/project/E3P8NfVEMc")
+            }
+        }
+
+        Column
+        {
```
Comment (Michal-Szczepaniak) [inline qml/pages/TranslationsPage.qml]:
```suggestion
        Column {
```

File: qml/pages/TranslationsPage.qml [hunk 17]
```diff
@@ -0,0 +1,56 @@
...
+            id: content
+            width: parent.width
+            spacing: Theme.paddingLarge
+
+            PageHeader
+            {
```
Comment (Michal-Szczepaniak) [inline qml/pages/TranslationsPage.qml]:
```suggestion
            PageHeader {
```

**Review by Michal-Szczepaniak — CHANGES_REQUESTED** [review]

**Review by Michal-Szczepaniak — APPROVED** [review]

**Pull request merged**